locale/cs/hammer_cli_foreman_ansible.po

```diff
@@ -0,0 +1,449 @@
+# Czech translations for hammer_cli_foreman_ansible package.
```

Comment:
You're adding this file, but it doesn't appear to include any translations. We normally only do this with > 50% translations.